src/tests/bll/services/test_sqlite_generator.py

```diff
@@ -13,89 +14,99 @@
 from tests.fake.fake_parser_mapping import FakeParserMapper
 from tests.fake.fake_sqlite_plugin_helper import FakeSQLitePluginHelper
 from tests.fake.fake_sqlite_plugin_path_helper import FakeSQLitePluginPathHelper
+from tests.test_helper import path_helper
 
 
 class SQLiteGeneratorTest(unittest.TestCase):
-  def setUp(self):
+  """the sqlite generator test"""
 
-    self.output_lambda = lambda x: self._writeToFile(x)
-    self.plugin_helper = FakeSQLitePluginHelper
+  def setUp(self):
+    self.plugin_helper = FakeSQLitePluginHelper()
     self.testfile = "temp"
+    self.template_path = path_helper.TemplatePath()
 
   def tearDown(self):
     if os.path.isfile(self.testfile):
       os.remove(self.testfile)
 
-  def test_print_copy(self):
+  def testPrintCopy(self):
+    """test the print for a copy"""
     with tempfile.TemporaryDirectory() as tmpdir:
-      path_helper = FakeSQLitePluginPathHelper
+      path_helper = FakeSQLitePluginPathHelper(self.template_path, 'test', 'db')
       file = os.path.join(tmpdir, 'testfile')
-      generator = SQLiteGenerator(tmpdir, 'test', 'test',['test'],
-        OutputHandlerFile(file, FileHandler), self.plugin_helper, path_helper)
-      generator._print_copy(file)
+      generator = SQLiteGenerator(tmpdir, 'test', 'test', ['test'],
+                                  OutputHandlerFile(file, FileHandler),
+                                  self.plugin_helper, path_helper)
+      generator._PrintCopy(file)  # pylint: disable=W0212
       expected = "copy " + file
-      actual = self._readFromFile(file)
+      actual = self._ReadFromFile(file)
     self.assertEqual(expected, actual)
 
-  def test_print_edit(self):
+  def testPrintEdit(self):
+    """test the print for a edit"""
     with tempfile.TemporaryDirectory() as tmpdir:
-      path_helper = FakeSQLitePluginPathHelper
+      path_helper = FakeSQLitePluginPathHelper(self.template_path,'test','db')
       file = os.path.join(tmpdir, 'testfile')
-      generator = SQLiteGenerator(tmpdir, 'test', 'test',['test'],
-        OutputHandlerFile(file, FileHandler), self.plugin_helper, path_helper)
-      generator._print_edit(file)
+      generator = SQLiteGenerator(tmpdir, 'test', 'test', ['test'],
+                                  OutputHandlerFile(file, FileHandler),
+                                  self.plugin_helper, path_helper)
+      generator._PrintEdit(file)  # pylint: disable=W0212
```

Comment:
instead of disable=W0212 use the descriptive name disable=protected-access

you can do the disable at class level seeing that tests are somewhat expected to access protected attributes and methods